### PR TITLE
Cronjobs_run_once_comanage_registry_role

### PR DESCRIPTION
**Pull request by ioigoume**
Cron jobs deploy only in one node

### CODE REVIEW

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
Introduce `comanage_cron_jobs_enabled: yes` in defaults (similar to https://github.com/rciam/rciam-deploy/blob/78e7f7fed2f63124f644f301a8e3a16b146c2d4f/roles/rciam-utils/defaults/main.yml#L52)
and set `comanage_cron_jobs_enabled` to False for node 2 in inventory

File: roles/comanage-registry/defaults/main.yml [hunk 1]
```diff
@@ -149,4 +149,7 @@ comanage_ldap_provisioner_entitlements:
 #    filename: "registry"
 #    user: root
 #    job: |
-#      cd {{ path_to_registry }}/app && Console/cake action
\ No newline at end of file
+#      cd {{ path_to_registry }}/app && Console/cake action
+
+# Whether to install RCIAM util cron jobs; Enabled by default
```
Comment (NicolasLiampotis) [inline roles/comanage-registry/defaults/main.yml:154]: Fix name in comment:
`# Whether to install COmanage Registry cron jobs; Enabled by default`

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

File: roles/comanage-registry/defaults/main.yml (hunk 1 above)
Comment (ioigoume) [reply roles/comanage-registry/defaults/main.yml]: Done

**Pull request merged**